smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/middleware/MiddlewareExecutionGenerator.kt

```diff
@@ -26,12 +28,18 @@ class MiddlewareExecutionGenerator(
     private val model: Model = ctx.model
     private val symbolProvider = ctx.symbolProvider
 
-    fun render(service: ServiceShape, op: OperationShape, onError: (SwiftWriter, String) -> Unit) {
```

Comment:
Add codegen for code that sets flags used by auth schemes in aws-sdk-swift to customize signing properties for presign url & presign request flows. 

E.g., `SigV4AuthScheme::customizeSigningProperties` in  aws-sdk-swift sets the type of its signature type (`.requestQueryParams` v. `.requestHeaders`) based on the `flowType` flag set on the middleware context.